src/App.js

```diff
@@ -8,7 +8,7 @@ function App() {
     <div className="App">
       <FormikLoginForm />
       <FormikRegisterForm />
-      <h1>WELCOME TO FORKBOOK</h1>
+      <h1 style={{ "fontSize": "1.8rem"}}> WELCOME TO FORKBOOK</h1>
```

Comment:
Even if it's just as a place holder please refrain from inlining the styles into an element, in case it gets lost or we forget to remove it